client/client_functions.py

```diff
@@ -176,19 +176,19 @@ def get_user_friends(server_url:str)->tuple:
         
     return tuple(usernames)
 
-def update_rating_for_thought(server_url:str, key:str)->None:
+def update_rating_for_thought(server_url:str, key:bytes):
     """function to update a thoughts rating when it is read"""
     account_url_suffix = "api/v1/update-thought-rating"
 
     headers = {"Authorization": f"Bearer {get_token()}"}
 
-    params = {"key" : key}
+    params = {"key": key}
     
-    response = requests.get(f"{server_url}{account_url_suffix}", headers=headers, params = params, timeout=10)
+    response = requests.get(f"{server_url}{account_url_suffix}", headers=headers, params=params["key"], timeout=10)
```

Comment:
Had to update this line in accordance to the return type screams

Reply:
that is kinda ridiculous lol... I wasn't even aware this was a thing in python